src/components/simple/SyntaxHighlightingInput.tsx

```diff
@@ -8,6 +8,8 @@ import "codemirror/addon/mode/simple";
 import "codemirror/addon/edit/matchbrackets";
 import "codemirror/addon/display/placeholder";
 
+const CODE_MIRROR_LINE_DELIMITER = "\n";
```

Comment:
I would rename it to `NEWLINE`, `END_OF_LINE`, `EOL` or something like that... It's not codemirror specific and it's not called line delimeter